samples/sample-production/shared/build.gradle.kts

```diff
@@ -14,22 +14,36 @@ plugins {
     kotlin("native.cocoapods")
 }
 
+android {
+    compileSdk = libs.versions.compileSdk.get().toInt()
+    defaultConfig {
+        minSdk = libs.versions.minSdk.get().toInt()
+        targetSdk = libs.versions.targetSdk.get().toInt()
+    }
+
+    val main by sourceSets.getting {
+        manifest.srcFile("src/androidMain/AndroidManifest.xml")
+    }
+}
+
 val KERMIT_VERSION: String by project
 
 kotlin {
     version = "0.0.1"
     android()
     ios()
+    // Note: iosSimulatorArm64 target requires that all dependencies have M1 support
+    iosSimulatorArm64()
```

Comment:
The M1 simulator target needs to also use the iosMain sourceset. See KaMP Kit for an example: https://github.com/touchlab/KaMPKit/blob/main/shared/build.gradle.kts#L103